appium-examples/src/test/java/com/native_app/biometric_login/README.md

```diff
@@ -0,0 +1,80 @@
+# Using biometric login on Sauce Labs
+This folder contains examples for using Biometric login on real devices for:
```

Comment:
@eyaly thanks for the wonderful docs. Please keep in mind that we need to maintain all of this and this is the #1 priority. We should avoid having duplicate instructions when we can. Hence, we don't need instructions on how to setup sauce env variables.

Reply:
Remove from the read me this part ( even that - for the users it is easy that they have image_injection readme with all the info and they don't need to read few readme files)